src/python/WMCore/MicroService/DataStructs/MSRuleCleanerWflow.py

```diff
@@ -7,6 +7,126 @@
 from __future__ import division, print_function
 
 from copy import deepcopy
+from itertools import chain
```

Comment:
Don't forget updating this to use the WMCore Utils package, as I mentioned in the previous review.

Reply:
Done